07.bowling_object/lib/shot.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+STRIKE = 10
+
+class Shot
+  attr_reader :mark
+
+  def initialize(mark)
+    @mark = mark
+  end
+
+  def score
+    return STRIKE if mark == 'X'
+
+    mark.to_i
+  end
+
+  def strike?
+    mark == 'X'
```

Comment:
```suggestion
    mark == STRIKE
```

むしろ定数化するならこっち？
でもこのままでも全然問題ないです。